src/routes/voteOnSponsorTime.ts

```diff
@@ -247,7 +247,8 @@ async function categoryVote(UUID: SegmentUUID, userID: UserID, isVIP: boolean, i
     const timeSubmitted = Date.now();
 
     const voteAmount = (isVIP || isTempVIP) ? 500 : 1;
-    const ableToVote = isVIP || isTempVIP || finalResponse.finalStatus === 200 || true;
```

Comment:
should we keep isVIP here?

Reply:
I don't think so, there are VIP checks above where needed. The only case where it would matter is if there was a (temp)VIP who was also banned at the same time.
finalStatus actually isn't modified anywhere in the code, it could be removed.